README.md

```diff
@@ -19,33 +19,68 @@ Composer will automatically install dependencies, register standards paths, and
 To include standards as part of a project require them as development dependencies:
 
 ```bash
-composer require yoast/yoastcs:dev-master --dev
+composer require --dev yoast/yoastcs:^1.0
 ```
 
-Note that Composer won't run configuration scripts in this scenario and the root project needs to take care of it.
+Composer will automatically install dependencies and register the YoastCS and other external standards with PHP_CodeSniffer.
 
 ## PHP Code Sniffer
 
 Set of [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) rules.
 
-Based on [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards) project, implementing official [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/php/).
-
 Severity levels:
 
  - error level issues are considered mandatory to fix in Yoast projects and enforced in continuous integration
  - warning level issues are considered recommended to fix
 
-### Command line
+### The YoastCS Standard
+
+The `Yoast` standard for PHP_CodeSniffer is comprised of the following:
+* The `WordPress` ruleset from the [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards) implementing the official [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/php/), with some select exclusions.
```

Comment:
Is there a way to link to (a list of) those select exclusions?

Reply:
Or maybe point to `#### Sniffs` below

Reply:
Not really. Only thing I can think of is linking to the `ruleset.xml` file which contains the excludes.